pkg/flare/config_check.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // ConfigCheckURL contains the Agent API endpoint URL exposing the loaded checks
-var ConfigCheckURL = fmt.Sprintf("https://localhost:%v/agent/config-check", config.Datadog.GetInt("cmd_port"))
+var ConfigCheckURL string
```

Comment:
let's remove this var as well?

(seems to be used in `pkg/flare/archive_test.go`, but I assume the test could be updated by simply setting `cmd_port`)

Reply:
@olivielpeau the test is done using `httptest` (https://golang.org/pkg/net/http/httptest/)  is it possible to have a custom port and url with this?

Reply:
Not sure if `httptest` is always on `localhost` but parsing the port from the URL seems to work 🤔

Reply:
In fact, I had to add it back because it was also used for to change the config check URL for the cluster agent (after rebase, I forgot about this) but I made it private